catalyst-gateway/tests/integration/catalyst_gateway_event_db.rs

```diff
@@ -6,18 +6,23 @@ async fn health_live() {
     let client = reqwest::Client::new();
     let url = format!("{cat_gateway_address}/api/health/live");
 
-    let response = unwrap!(
-        client.get(&url).send().await,
-        "Failed to execute request to {url}."
-    );
-    assert_eq!(
-        response.status().as_u16(),
-        204,
-        "request failed for {url} response: {response:#?}"
-    );
-    assert_eq!(
-        Some(0),
-        response.content_length(),
-        "request failed for {url} response: {response:#?}"
-    );
+    let response = client.get(&url).send().await;
```

Comment:
No, it is better to leave a previous example with `unwrap!(client.get(&url).send().await,"Failed to execute request to {url}.")`. Because it checks that request was successful and fails if not.
It you implementation you are not failing when request is not successful, it just prints an error to `stderr`

Reply:
Maybe I will throw panic instead of printing the error. Got a error from linter when using unwrap though

Reply:
If using `unwrap!` causing problems for us, for such places use `expect()` at the same place where you are checking is everything ok or not.
```rust
let response = client.get(&url).send().await.expect("Failed to execute request to {url}.");
```